src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,42 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type PostProp = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<PostProp> = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by  '}
+
+          {user && (<UserInfo user={user} />)}
+        </p>
+      </div>
+
+      <p className="PostInfo__body">
+        {body}
+      </p>
+
+      <hr />
+
+      {comments.length > 0
+        ? <CommentList comments={comments} />
+        : <b data-cy="NoCommentsMessage">No comments yet</b>}
```

Comment:
```suggestion
        : <b data-cy="NoCommentsMessage">No comments yet</b>
      }
```

Reply:
fix formatting here, as Serhii suggested

Reply:
Linter doesn't let me do it as you write. It automatically fixed it.